CharacterCreation/Class-Specific Documentation/Engineer Processes.txt

```diff
@@ -1179,13 +1289,20 @@ Size:				1m x .6m x .8m
 Disrupt DC:			60
 Pwn DC:				90
 
++-------+------+------+-----+
+| Range |  3m  | 10m  | 20m |
++-------+------+------+-----+ 
+| Damage|  80  |  65  |  50 |   
++-------+------+------+-----+
+| Acc   |   2  |   4  |  5  | 
++-------+------+------+-----+
+
     You construct a quadrupedal drone and upload the canine AI to it's 
 processor. You imprint your team's hunter upon it. The pointer comes 
 prefabricated with a head-mounted 12-gauge peripheral with 10 shots. It also has 
```

Comment:
Looks more like a 24-gauge from the damage output, and the shotgun ballistics are much worse than a typical player shotgun. (It looks like you used the older version of shotgun ballistics, with the 3-step damage instead of the 2 step)
If that's what you want, then this works for me. Non-blocking

Reply:
It's a custom shotgun, not a 24 gauge